app/main/forms.py

```diff
@@ -0,0 +1,85 @@
+from flask.ext.wtf import Form
+from wtforms.fields import StringField, SubmitField, IntegerField, TextAreaField, HiddenField, \
+    DateField, FileField
+from ..models import Agency
+from wtforms.ext.sqlalchemy.fields import QuerySelectField
+from wtforms.validators import (
+    InputRequired,
+    Length,
+    Optional,
+    NumberRange,
+    URL
+)
+from app.custom_validators import StrippedLength
+from .. import db
+import datetime as datetime
+
+
+class IncidentReportForm(Form):
```

Comment:
I think I merged master's and also have reports/forms.py. I guess I'll just remove this file then?